storage/innobase/dict/dict0stats.cc

```diff
@@ -2317,17 +2434,20 @@ dict_stats_index_set_n_diff(
 /** Calculates new statistics for a given index and saves them to the index
 members stat_n_diff_key_vals[], stat_n_sample_sizes[], stat_index_size and
 stat_n_leaf_pages. This function can be slow.
-@param[in]	index	index to analyze
+@param[in]	index	      index to analyze
+@param[in]      stats_method  innodb_stats_method variable
 @return index stats */
-static index_stats_t dict_stats_analyze_index(dict_index_t* index)
+static index_stats_t dict_stats_analyze_index(dict_index_t* index,
+                                              unsigned stats_method)
 {
 	bool		level_is_analyzed;
+	bool		nulls_unequal = (stats_method > SRV_STATS_NULLS_EQUAL);
+	bool		calc_non_null =
+		(stats_method == SRV_STATS_NULLS_IGNORED);
 	ulint		n_uniq;
 	ulint		n_prefix;
-	ib_uint64_t	total_recs;
-	ib_uint64_t	total_pages;
 	mtr_t		mtr;
-	index_stats_t	result(index->n_uniq);
+	index_stats_t 	result(index->n_uniq);
```

Comment:
This unnecessary change seems to add an "invisible" space before a TAB character.